roles/tas_single_node/README.md

```diff
@@ -79,6 +82,22 @@ Deploy the [RHTAS](https://docs.redhat.com/en/documentation/red_hat_trusted_arti
 | root_password | The root password for the database. | str | no |  |
 | database | The database name to connect to. | str | no |  |
 
+#### Options for main > tas_single_node_fulcio
+
+|Option|Description|Type|Required|Default|
+|---|---|---|---|---|
+| certificate | Details on the certificate attributes for Fulcio. | dict of 'certificate' options | no |  |
+| tas_single_node_fulcio_private_key | The user-provided private key for Fulcio, used for signing root certificate (only RSA with > 2048 B pkey size or ECC with prime256v1 (==secp256r1) are supported). | str | no |  |
+| tas_single_node_fulcio_root_ca | The user-provided root certificate for Fulcio. | str | no |  |
```

Comment:
Let's mention that this is mutually exclusive with `certificate` (it is, right?)